package.json

```diff
@@ -48,27 +44,21 @@
     ]
   },
   "devDependencies": {
-    "@types/jest": "^27.0.0",
-    "@types/node": "^16.4.14",
-    "@typescript-eslint/eslint-plugin": "^5.42.0",
-    "@typescript-eslint/parser": "^5.42.0",
-    "@vuepress/plugin-back-to-top": "^1.9.7",
-    "@vuepress/plugin-medium-zoom": "^1.9.7",
-    "eslint": "^8.26.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "husky": "^7.0.4",
-    "jest": "^27.0.6",
-    "lint-staged": "^11.1.2",
-    "prettier": "^2.7.1",
-    "pretty-quick": "^3.1.1",
-    "rimraf": "^3.0.2",
-    "ts-jest": "^27.0.4",
-    "typedoc": "^0.19.2",
-    "typedoc-plugin-markdown": "^2.4.2",
-    "typescript": "^4.8.4",
-    "vuepress": "^1.9.7",
-    "vuepress-plugin-one-click-copy": "^1.0.6"
+    "@types/jest": "27.0.0",
+    "@types/node": "16.4.14",
+    "@typescript-eslint/eslint-plugin": "5.42.0",
+    "@typescript-eslint/parser": "5.42.0",
+    "eslint": "8.26.0",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-prettier": "4.2.1",
+    "husky": "7.0.4",
+    "jest": "27.0.6",
+    "lint-staged": "11.1.2",
+    "prettier": "2.7.1",
+    "pretty-quick": "3.1.1",
+    "rimraf": "3.0.2",
+    "ts-jest": "27.0.4",
+    "typescript": "4.8.4"
```

Comment:
I think we want to allow for minor version updates, right?

Reply:
no, from previous experience we agreed to remove the ^, installing exactly the package version we are using due to many nodejs packages don't respect the SEMVER, and releasing breaking changes under a minor version.

Reply:
I think that is fair considering these are devDependencies.  I think others, especially the Couchbase SDK, we should allow for minor bumps.